src/topology/sheaves/stalks.lean

```diff
@@ -109,6 +132,28 @@ lemma stalk_hom_ext (F : X.presheaf C) {x} {Y : C} {f₁ f₂ : F.stalk x ⟶ Y}
   (ih : ∀ (U : opens X) (hxU : x ∈ U), F.germ ⟨x, hxU⟩ ≫ f₁ = F.germ ⟨x, hxU⟩ ≫ f₂) : f₁ = f₂ :=
 colimit.hom_ext $ λ U, by { op_induction U, cases U with U hxU, exact ih U hxU }
 
+/-- If two sections agree on all stalks, they must be equal -/
+lemma section_ext (F : sheaf (Type v) X) (U : opens X) (s t : F.presheaf.obj (op U)) :
```

Comment:
Should this be an `@[ext]` lemma too? I feel like this would apply in too general situations and showing equality of sections stalkwise doesn't seem like something one would want to do very often, so I left it out,

Reply:
I agree - no `@[ext]` here.